src/backend/renderer/mod.rs

```diff
@@ -168,26 +168,62 @@ pub trait Frame {
         &mut self,
         texture: &Self::TextureId,
         matrix: Matrix3<f32>,
+        verticies: [Vector2<f32>; 4],
         alpha: f32,
     ) -> Result<(), Self::Error>;
+
     /// Render a texture to the current target as a flat 2d-plane at a given
-    /// position, applying the given transformation with the given alpha value.
+    /// position, cropping at the given offsets and applying the given transformation with the given alpha value.
```

Comment:
It doesn't look like this method does any cropping, it does not take any argument indicating it.